### PR TITLE
🐛 Fix `HoVer-Net` Class Output Information in `docs`

### PR DESCRIPTION
**Pull request by adamshephard**
- [X] Fix bug with HoVer-Net, where currently merging classes 0 and 1. This is resulting in the background class being omitted and merged with the first.
- [x] Add the output nuclear classes for the nuclear instance segmentation models to the documentation. We have also added type dictionaries to the pretrained_yaml file.
- [x] Update notebooks to show this correction.

https://tia-toolbox.readthedocs.io/en/add-model-docs/

### CODE REVIEW

**Comment by shaneahmed** [comment]
It's failing due to an update in pretrained_models.yaml.

File: docs/pretrained.rst [hunk 1]
```diff
@@ -185,6 +185,15 @@ The input output configuration is as follows:
 
     - hovernet_fast-pannuke
 
+.. collapse:: Output Nuclear Classes
+
+    - 0: Background
```
Comment (shaneahmed) [inline docs/pretrained.rst:190]: Does this class exist in the TIAToolbox version?

**Comment by Taylor-Kelce** [comment]
Is there a reason why micronet_hovernet-consep is not not available for Nucleus instance segmentor? 

```python
inst_segmentor = NucleusInstanceSegmentor(
    pretrained_model='micronet_hovernet-consep',
    num_loader_workers=4,
    num_postproc_workers=4,
    batch_size=10,
)
```

File: docs/pretrained.rst (hunk 1 above)
Comment (adamshephard) [reply docs/pretrained.rst:190]: No- changing

**Comment by shaneahmed** [comment]
> Is there a reason why micronet_hovernet-consep is not not available for Nucleus instance segmentor?
> 
> ```python
> inst_segmentor = NucleusInstanceSegmentor(
>     pretrained_model='micronet_hovernet-consep',
>     num_loader_workers=4,
>     num_postproc_workers=4,
>     batch_size=10,
> )
> ```

MicroNet is semantic segmentation only, it is not trained for instance segmentation.

File: docs/pretrained.rst [hunk 2]
```diff
@@ -275,11 +300,20 @@ The input output configuration is as follows:
 
     - micronet_hovernet-consep
 
+.. collapse:: Output Nuclear Classes
```
Comment (shaneahmed) [inline docs/pretrained.rst:303]: I think micronet is just a segmentation model. No classification.

**Review by shaneahmed — CHANGES_REQUESTED** [review]
Thanks @adamshephard Just a minor comment, I can merge it after this has been addressed.

File: docs/pretrained.rst (hunk 2 above)
Comment (adamshephard) [reply docs/pretrained.rst:303]: Edited - thanks.

Comment (adamshephard) [reply docs/pretrained.rst:303]: Edited thanks

**Pull request merged**